wcfsetup/install/files/ts/WoltLabSuite/Core/Form/Builder/Field/Tag.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Data handler for a tag form builder field in an Ajax form.
+ *
+ * @author  Matthias Schmidt
+ * @copyright 2001-2020 WoltLab GmbH
+ * @license GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Form/Builder/Field/Tag
+ * @since 5.2
+ */
+
+import Field from "./Field";
+import * as UiItemList from "../../../Ui/ItemList";
+import { FormBuilderData } from "../Data";
+import * as Core from "../../../Core";
+
+class Tag extends Field {
+  protected _getData(): FormBuilderData {
+    const values: string[] = UiItemList.getValues(this._fieldId).map((item) => {
+      return item.value;
+    });
```

Comment:
```suggestion
    const values: string[] = UiItemList.getValues(this._fieldId).map((item) => item.value);
```
This is simple enough to not need the braces. Personal preference though. If you want to apply this then make sure to do this locally as I assume prettier will insert linebreaks afterwards.